app/assets/javascripts/groups_index.js

```diff
@@ -37,6 +40,26 @@ if (!top.location.pathname.includes("users")) {
     }
   }
 
+  function leave_group(group_id) {
+    $button.removeClass("is-member");
+    $button.removeClass("remove-member");
+    $button.text("Join Group");
+    $.ajax({
+      type: "PUT",
+      url: "/groups/" + $button.attr("value") + "/leave",
+      success: function(data) {
+        $(".group_id" + group_id).removeClass("active");
+        $(".group_id" + group_id).removeClass("inactive");
+        $(".group_id" + group_id).addClass("" + data.state);
```

Comment:
you can condense these three lines to `$div.removeClass("active inactive").addClass(data.state)`